src/SprykerMiddleware/Zed/Process/Business/Mapper/Mapper.php

```diff
@@ -9,14 +9,11 @@
 
 use Closure;
 use Generated\Shared\Transfer\MapperConfigTransfer;
-use SprykerMiddleware\Shared\Logger\Logger\MiddlewareLoggerTrait;
 use SprykerMiddleware\Zed\Process\Business\ArrayManager\ArrayManagerInterface;
 use SprykerMiddleware\Zed\Process\Business\Mapper\Map\MapInterface;
 
 class Mapper implements MapperInterface
 {
-    use MiddlewareLoggerTrait;
```

Comment:
Please restore here and below MiddlewareLoggerTrait
use trait from new module spryker-middleware/logger